core/ai.go

```diff
@@ -127,3 +132,102 @@ func ParseStepsFromModelID(modelID *string, defaultSteps float64) float64 {
 
 	return numInferenceSteps
 }
+
+// AddAICapabilities adds AI capabilities to the node.
+func (n *LivepeerNode) AddAICapabilities(caps *Capabilities) {
```

Comment:
This function should not exist

Reply:
Not following. Is this not similar to `AddCapacity`?

You are saying it would be better to squash this into `AddCapacity`? Without updating tests to cover this addition into `AddCapacity` function?

New function was added because transcoding does not use capabilities in this way right now and tried to avoid adding complication to a function used with remote transcoder connection that didn't need to be there.